### PR TITLE
FALCON-1842 Falcon build failed in Jenkins at OozieFeedWorkflowBuilde…

### PR DESCRIPTION
**Pull request by bvellanki**
…rTest (Balu Vellanki)

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -35,6 +35,8 @@ Trunk
     FALCON-1770 Update README file (Ajay Yadava)
 
   BUG FIXES
+    FALCON-1842 Falcon build failed in Jenkins at OozieFeedWorkflowBuilderTest (Balu Vellanki)
```
Comment (pavankumar526) [inline CHANGES.txt]: Sometimes we are adding and sometimes not. @pallavi-rao  @sandeepSamudrala  What is the final call on this

Comment (pallavi-rao) [inline CHANGES.txt]: CHANGES.txt doesn't need to be added. As we can have a script to generate the change log, PR 34

**Comment by pallavi-rao** [comment]
+1 for the patch.

**Comment by ajayyadava** [comment]
+1. Will commit shortly.

**Pull request closed without merging**